metadata-ingestion/src/datahub/cli/docker_cli.py

```diff
@@ -670,6 +718,12 @@ def quickstart(
     auth_resources_folder = Path(DATAHUB_ROOT_FOLDER) / "plugins/auth/resources"
     os.makedirs(auth_resources_folder, exist_ok=True)
 
+    stable = not unstable
+    if stable:
```

Comment:
If unstable, where does the code pick up the compose files from?

Reply:
Can we still use DOCKER_COMPOSE_BASE env var to specify where to get the compose files from?
We use it in a few places:
- https://github.com/datahub-project/datahub/blob/7cb3d7209cb5219dd4ae149809287fb6e0725a94/smoke-test/run-quickstart.sh#L17
- https://github.com/datahub-project/datahub/blob/7cb3d7209cb5219dd4ae149809287fb6e0725a94/docker/build.gradle#L25
- https://github.com/datahub-project/datahub/blob/7cb3d7209cb5219dd4ae149809287fb6e0725a94/metadata-integration/java/spark-lineage/spark-smoke-test/smoke.sh#L43